src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -230,13 +230,11 @@ const PlrDaoStakingTransactionBlock = ({
   };
 
   const getBalanceForAllChains = async () => {
-    const chainPromise: AccountBalance[] = [];
-    supportedChains.forEach(async (chain) => {
-      chainPromise.push(await getWalletBalance(chain.chainId, chain.title));
-    });
-    return Promise.allSettled(chainPromise).catch((e) => {
+    try {
+      return Promise.allSettled(supportedChains.map(async (chain) => getWalletBalance(chain.chainId, chain.title)));
```

Comment:
I was asking in previous PR to remove `allSettlet` with `fulfilled` into just `Promise.all` approach as it straight forward and you don't need fulfillment checks of your promises

Reply:
@poocart Yes. In `Promise.all` case, when we get error in any of chain, it will also reject other chains Promises and not able to display PLR tokens count.
If We use `Promise.allSettled`, we can atleast display the token for other Chains, even if we get error for any of the chain.

Reply:
@poocart Updated!